code/modules/station_goals/dna_vault.dm

```diff
@@ -219,5 +219,6 @@
 		if(VAULT_QUICK)
 			to_chat(H, span_notice("Your arms move as fast as lightning."))
 			H.next_move_modifier = 0.5
+	ADD_TRAIT(H, TRAIT_USED_DNA_VAULT, "used_dna_vault")
```

Comment:
ADD_TRAIT() has 3 args, H is what we're adding the trait to, ``TRAIT_USED_DNA_VAULT`` is the trait we're giving, and ``"used_dna_vault"`` is the source trait, where the trait is coming from.

Trait stuff including the source trait should be defines, so like how you made a define for ``TRAIT_USED_DNA_VAULT``, you should make a new one called ``DNA_VAULT_TRAIT``.

Reply:
thanks for clarification

Reply:
you mean here? 
![image](https://user-images.githubusercontent.com/85910816/180119555-512d5df6-487c-479a-8044-a5e5f24e5409.png)